rust/connlib/clients/apple/src/lib.rs

```diff
@@ -96,7 +96,7 @@ pub struct CallbackHandler {
     // refcount, but there's no way to generate a `Clone` impl that increments the
     // recount. Instead, we just wrap it in an `Arc`.
     inner: Arc<ffi::CallbackHandler>,
-    handle: file_logger::Handle,
+    _handle: file_logger::Handle,
```

Comment:
For consistency, we could store this in the `WrappedSession` same as the `Runtime`.

Reply:
👍